react/features/conference/components/native/constants.js

```diff
@@ -0,0 +1,125 @@
+import { TransitionPresets } from '@react-navigation/stack';
+import React from 'react';
+import { Platform, TouchableWithoutFeedback } from 'react-native';
+
+import { Icon, IconArrowBack } from '../../../base/icons';
+
+import * as ConferenceNavigationContainerRef
+    from './ConferenceNavigationContainerRef';
+
+
+/**
+ * Default modal transition for the current platform.
+ */
+export const conferenceModalPresentation = Platform.select({
+    ios: TransitionPresets.ModalPresentationIOS,
+    default: TransitionPresets.DefaultTransition
+});
+
+/**
+ * Screen options and transition types.
+ */
+export const screenOptions = {
+    ...TransitionPresets.ModalTransition,
+    gestureEnabled: false,
+    headerShown: false
+};
+
+/**
+ * Screen options for conference.
+ */
+export const conferenceScreenOptions = {
+    ...screenOptions
+};
+
+/**
+ * Screen options for lobby modal.
+ */
+export const lobbyScreenOptions = {
+    ...screenOptions
+};
+
+/**
+ * Tab bar options for chat screen.
+ */
+export const chatTabBarOptions = {
+    activeTintColor: '#17a0db',
```

Comment:
Let's have tokens for these please.

Reply:
I looked into it and it seems that I cannot there are only two ways to use it.
![image](https://user-images.githubusercontent.com/26413496/135407390-1bd72492-7aa6-42b0-b1a8-48249292101e.png)

One solution would be to create some variables in Tokens.js file and export them without using theme

Reply:
Let's use the tokens approach then. At the end of the day that's how the Theme gets built...

Reply:
Updated